optd-persistent/src/migrator/catalog/m20241029_000001_column.rs

```diff
@@ -1,10 +1,8 @@
-use sea_orm_migration::prelude::*;
-use sea_orm_migration::schema::*;
-
-use super::table_metadata::TableMetadata;
+use crate::migrator::catalog::table_metadata::TableMetadata;
+use sea_orm_migration::{prelude::*, schema::*};
 
 #[derive(Iden)]
-pub enum TableAttribute {
+pub enum Column {
```

Comment:
I believe @apavlo wants to follow postgres convention which uses Attribute instead of Column (primarily for historical reasons)

Reply:
My opinion is that we should prefer clarity over postgres conventions: When you say column, everyone knows exactly what you mean. When you say attribute, most people know what you mean, but some people might not and some people might even misinterpret.

Reply:
Discussed in meeting, changing back to attribute in [1807cc1](https://github.com/cmu-db/optd-experimental/pull/10/commits/1807cc1dc415eb6e9bf2fe595aa207ab5a6be23f)